tests/core/pyspec/eth2spec/test/lightclient_patch/block_processing/test_process_sync_committee.py

```diff
@@ -72,12 +74,18 @@ def compute_sync_committee_participant_reward(spec, state, participant_index, ac
 
 
 @with_all_phases_except([PHASE0, PHASE1])
+@with_configs([MINIMAL], reason="to create nonduplicate committee")
 @spec_state_test
-def test_sync_committee_rewards(spec, state):
+def test_sync_committee_rewards_nonduplicate_committee(spec, state):
     committee = spec.get_sync_committee_indices(state, spec.get_current_epoch(state))
     committee_size = len(committee)
     active_validator_count = len(spec.get_active_validator_indices(state, spec.get_current_epoch(state)))
 
+    # Preconditions of this test case
+    # Note that the committee members MAY still be duplicate even with enough active validator count probabilistically.
+    assert active_validator_count >= spec.SYNC_COMMITTEE_SIZE
+    assert committee_size == len(set(committee))
```

Comment:
@djrtwo @ralexstokes
This is a simple fix by only using minimal config.
Note that it's `SYNC_COMMITTEE_SIZE=32` and `active_validator_count = 64`, but it's still possible to have duplicate elements probabilistically.

I'm not fully happy about the probabilistic cases and keen to see if there is another better way to generate a 100% nonduplicate indices case.

Reply:
You can do a `while` loop, changing the `seed` (directly manipulating the randao mix) until this condition is true.

It might save a headache in the future. Your call